strings/base_activation.h

```diff
@@ -257,8 +257,14 @@ namespace winrt::impl
         }
 
     private:
-
-        size_t& m_count;
+#ifdef __clang__
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunused-private-field"
+#endif
+        size_t& m_count; // Field is unused when WINRT_NO_MODULE_LOCK is defined.
+#ifdef __clang__
+#pragma clang diagnostic pop
+#endif
```

Comment:
So apparently you can use `[[maybe_unused]]` on a private data member: https://godbolt.org/z/Gq5qbKMKz. Looks a bit cleaner

Reply:
Oh yeah, that's definitely better.  I tried "using" it in the `else` block above but the compiler was smart enough to optimize the NOOP usage out and still warn that it was unused.